src/cow-react/modules/limitOrders/pure/ReceiptModal/styled.ts

```diff
@@ -0,0 +1,145 @@
+import styled from 'styled-components/macro'
+import { AutoRow } from 'components/Row'
+
+export const Wrapper = styled.div`
+  border-radius: 16px;
+  background: ${({ theme }) => theme.bg1};
+  width: 100%;
+  overflow-y: hidden;
+  padding-bottom: 1.2rem;
+
+  ${({ theme }) => theme.mediaWidth.upToMedium`
+    max-height: 80vh;
+  `}
+
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    margin: 70px 0;
+    overflow-y: auto;
+    padding-bottom: 0;
+  `}
+`
+
+export const Header = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+`
+
+export const Title = styled.h3`
+  font-size: 1rem;
+  margin: 0;
+`
+
+export const Body = styled(AutoRow)`
+  box-sizing: border-box;
+  max-height: 700px;
+
+  ${({ theme }) => theme.mediaWidth.upToMedium`
+    max-height: 70vh;
+  `}
+
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    max-height: 100%;
+  `}
+`
+
+export const Field = styled.div<{ border?: 'rounded-full' | 'rounded-top' | 'rounded-bottom' }>`
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+  background: ${({ theme }) => theme.input.bg1};
+  margin-bottom: 5px;
+  width: 100%;
+  border-radius: ${({ border }) => {
```

Comment:
You can wrap all fields into one `<div>` and set border-radius: 1rem to the wrapper